### PR TITLE
add example on how to add correlation_id in uvicorn access log

### LINKED ISSUE #76: question: how to add correllation_id in http access log ?
by chance, do you have an example using _correllation_id_ in _uvicorn_ access log ?

### PR DESCRIPTION
**Pull request by nesies**
Closes #76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -468,6 +468,70 @@ priority_settings = {
...
+    # add correlation_id in uvicorn access_log
+    LOGGING_CONFIG["handlers"]["access"]["filters"] = [
+            asgi_correlation_id.CorrelationIdFilter()]
+    LOGGING_CONFIG["formatters"]["access"]["fmt"] = \
+        "%(asctime)s %(levelname)s %(correlation_id)s" \
+        " access %(client_addr)s - %(request_line)s %(status_code)s"
+    uvicorn.run(
+        "test:app",
+        port=8080,
+        log_level=os.environ.get("LOGLEVEL", "DEBUG").lower())
```
Comment (sondrelg) [inline README.md:532]: This looks good, but how about we make it a little simpler?
```suggestion
import logging
import os

import asgi_correlation_id
import uvicorn
from fastapi import APIRouter, FastAPI
from uvicorn.config import LOGGING_CONFIG


def configure_logging():
    console_handler = logging.StreamHandler()
    console_handler.addFilter(asgi_correlation_id.CorrelationIdFilter())
    logging.basicConfig(
        handlers=[console_handler],
        level=getattr(logging, os.environ.get("LOGLEVEL", "DEBUG")),
        format="%(levelname)s log [%(correlation_id)s] %(name)s %(message)s"
    )


app = FastAPI(on_startup=[configure_logging])
app.add_middleware(asgi_correlation_id.CorrelationIdMiddleware)
router = APIRouter()


@router.get("/test")
async def test_get():
    logger = logging.getLogger()
    logger.info("foo")


app.include_router(router)

if __name__ == "__main__":
    LOGGING_CONFIG["handlers"]["access"]["filters"] = [asgi_correlation_id.CorrelationIdFilter()]
    LOGGING_CONFIG["formatters"]["access"]["fmt"] = "%(levelname)s access [%(correlation_id)s] %(name)s %(message)s"
    uvicorn.run("app:app", port=8080, log_level=os.environ.get("LOGLEVEL", "DEBUG").lower())
```

File: README.md [hunk 2]
```diff
@@ -468,6 +468,70 @@ priority_settings = {
 }
 ```
 
+# Integration with [Uvicorn](https://github.com/encode/uvicorn)
+[Uvicorn](https://github.com/encode/uvicorn) is one of the ASGI servers
+used to run a [FastAPI](https://fastapi.tiangolo.com/deployment/manually/) application.
```
Comment (sondrelg) [inline README.md:473]:
```suggestion
To add a correlation ID to your [uvicorn](https://github.com/encode/uvicorn) logs, you'll need to add a log filter and change the log formatting. Here's an example of how to configure uvicorn, if you're running a [FastAPI](https://fastapi.tiangolo.com/deployment/manually/) app:
```

**Review by sondrelg — COMMENTED** [review]
Thanks for the PR! I've made two suggestions to (in my mind) streamline the explanation a bit :+1:

File: README.md (hunk 2 above)
Comment (nesies) [reply README.md]: updated thx

**Comment by sondrelg** [comment]
Thanks @nesies!

**Pull request merged**